fashion-example-query/requirements.txt

```diff
@@ -1,2 +1,6 @@
-jina[devel,torch,tensorflow,transformers]==0.7.12
 click
```

Comment:
Specific version

Reply:
I think I already changed this on the latest commit

Reply:
I don't see it for `click`.